src/routes/gifs.js

```diff
@@ -104,17 +105,37 @@ const createComment = catchAsync( async (req, res) => {
 
 })
 
+const recordLikesOnGif = catchAsync( async(req, res) => {
```

Comment:
let's keep the name singular since its just one happening at a time. `likeGif` is fine as the name

Reply:
yes please